src/SDKs/Blueprint/Management.Blueprint/Microsoft.Azure.Management.Blueprint.csproj

```diff
@@ -6,17 +6,16 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.Blueprint</PackageId>
     <Description>Provides Microsoft Azure Blueprint management operations including the ability to create, update and delete Blueprint definition and assignment.</Description>
-    <Version>0.12.0-preview</Version>
+    <Version>0.13.0-preview</Version>
     <!-- Remove the Preview description in PackageReleaseNotes once the preview is removed.-->
     <AssemblyTitle>Microsoft Azure Blueprint Management Library</AssemblyTitle>
     <AssemblyName>Microsoft.Azure.Management.Blueprint</AssemblyName>
     <PackageTags>Blueprint;</PackageTags>
     <PackageReleaseNotes>
 <![CDATA[This is a preview release sdk for Azure Blueprints, allows you to do Blueprint definition and assignment CRUD.
 
-Changes in 0.12.0-preview:
- - Adding support for excluding AAD principals from blueprint locks 
- - Adding support to retrieve blueprints service SPN objectID
```

Comment:
Please update the `AssemblyVersion` and `AssemblyFileVersion` in the `AssemblyFile.cs` file

Reply:
Updated.